### PR TITLE
[DOC-2752] Fixed the indentation issues in the March 8 delegate relea…

### PR DESCRIPTION
**Pull request by deepakbolangady**
…se notes (what's new topic - missed it earlier).

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/whats-new.md [hunk 1]
```diff
@@ -97,19 +97,16 @@ These changes are backward incompatible. Therefore, you must also update the Ter
...
-  - Custom - Create your own YAML from a [Kubernetes Manifest template](https://github.com/harness/delegate-kubernetes-manifest/blob/main/harness-delegate.yaml) by replacing the placeholders with the values provided in the method.
-  
- Given the need to have a fully qualified YAML, the auto upgrade configuration is set to ON in both the above options. The result is that the delegate version installed will always remain in sync with the version available on the Harness Manager.
- 
- - The **Docker** delegate installation method has now been enhanced to a simple copy and paste of the Docker run command, with the option to modify the pre-populated properties. The auto upgrade is set to OFF for this method with an option to enable it in the command. (DEL-6037)
+- The delegate installation UI experience is now updated with a new installation method: the Terraform Helm provider. Also, the installation experience has been enhanced for the existing methods (Helm chart, Kubernetes manifest, and Docker). This new experience is more developer friendly. For example, it enables cutting and pasting of relevant commands. You can also automate the commands and use new values when necessary.
```
Comment (douglas-j-bothwell) [inline release-notes/whats-new.md:100]:
```suggestion
- The delegate installation UI experience is now updated with a new installation method: the Terraform Helm provider. Also, the installation experience has been enhanced for the existing methods (Helm chart, Kubernetes manifest, and Docker). This new experience is more developer-friendly. For example, it enables cutting and pasting of relevant commands. You can also automate the commands and use new values when necessary. 
```

**Review by douglas-j-bothwell — APPROVED** [review]

File: release-notes/whats-new.md [hunk 2]
```diff
@@ -97,19 +97,16 @@ These changes are backward incompatible. Therefore, you must also update the Ter
...
+
+  Additionally, the following new features are available:
+    - The **Terraform Helm Provider** method is based on the open source [Terraform Harness Delegate module](https://registry.terraform.io/modules/harness/harness-delegate/kubernetes/latest) and the open source [Harness Delegate Helm chart](https://github.com/harness/delegate-helm-chart). Auto upgrade is set to `OFF` with an option to enable it in the command.
+    - The updated method for **Helm Chart** is also based on the same open source [Harness Delegate Helm chart](https://github.com/harness/delegate-helm-chart) as the Terraform Helm provider. Auto upgrade is set to OFF with an option to enable it in the command. You can also download the [default values.yaml](https://github.com/harness/delegate-helm-chart/blob/main/harness-delegate-ng/values.yaml) for the Helm option and edit that to set your own long-lived configuration values.
+    - The updated flow for **Kubernetes Manifest** has the following options for creating a manifest YAML file specific to your Harness account.
```
Comment (aimurphy) [inline release-notes/whats-new.md:105]:
```suggestion
    - The updated flow for **Kubernetes Manifest** has the following options for creating a manifest YAML file specific to your Harness account:         
```

Comment (aimurphy) [reply release-notes/whats-new.md:105]: May want to check the indentation on the build preview to make sure the bullets are at the proper level.

File: release-notes/whats-new.md [hunk 3]
```diff
@@ -97,19 +97,16 @@ These changes are backward incompatible. Therefore, you must also update the Ter
...
+    - The updated flow for **Kubernetes Manifest** has the following options for creating a manifest YAML file specific to your Harness account.          
+      - **Basic**: Provides a **Download YAML** option. The downloaded YAML has all the configuration variables set to values that are specific to your Harness account. 
+      - **Custom** - Create your own YAML from a [Kubernetes manifest template](https://github.com/harness/delegate-kubernetes-manifest/blob/main/harness-delegate.yaml) by replacing the placeholders with the values provided in the method.
+      Given the need to have a fully qualified YAML, the auto upgrade configuration is set to ON in both the above options. Consequently, the delegate version that is installed always remains in sync with the version available on Harness Manager.
```
Comment (aimurphy) [inline release-notes/whats-new.md:108]:
```suggestion
      Given the need to have a fully-qualified YAML, the auto upgrade configuration is set to `ON` in both of the above options. Consequently, the delegate version that is installed always remains in sync with the version available on Harness Manager.
```

File: release-notes/whats-new.md [hunk 4]
```diff
@@ -97,19 +97,16 @@ These changes are backward incompatible. Therefore, you must also update the Ter
...
+    - The **Docker** delegate installation method has now been simplified to a copy-and-paste action on the `docker run` command, with the option to modify the preset values. The auto upgrade is set to OFF for this method, with an option to enable it in the command. (DEL-6037)
```
Comment (aimurphy) [inline release-notes/whats-new.md:109]:
```suggestion
    - The **Docker** delegate installation method has now been simplified to a copy-and-paste action on the `docker run` command, with the option to modify the preset values. Auto upgrade is set to `OFF` for this method, with an option to enable it in the command. (DEL-6037)
```

**Review by aimurphy — APPROVED** [review]
approved w some suggestions

**Pull request merged**